spec/controllers/meetings_controller_spec.rb

```diff
@@ -22,8 +22,9 @@
     it 'makes a call to the MeetingStarter service with the right values' do
       logout = 'http://example.com'
       request.env['HTTP_REFERER'] = logout
+      url = nil
 
-      expect(MeetingStarter).to receive(:new).with(room:, logout_url: logout)
+      expect(MeetingStarter).to receive(:new).with(room:, logout_url: logout, url:)
 
       post :start, params: { friendly_id: room.friendly_id }
```

Comment:
You're missing a test case here. You need to test the case where you actually attach a real file to a room and make sure it gets passed through